cylc/flow/scheduler_cli.py

```diff
@@ -108,18 +108,6 @@ def get_option_parser(is_restart, add_std_opts=False):
         help="Do not daemonize the suite (infers --format=plain)",
         action="store_true", dest="no_detach")
 
-    parser.add_option(
-        "-a", "--no-auto-shutdown", help="Do not shut down"
-        " the suite automatically when all tasks have finished."
-        " This flag overrides the corresponding suite config item.",
-        action="store_true", dest="no_auto_shutdown")
```

Comment:
Have I assumed too much in removing the CLI option as well?

Reply:
No, I don't think so.